kubernetes/linux/main.sh

```diff
@@ -814,6 +816,8 @@ else
             echo "export MDSD_FLUENT_SOCKET_PORT=$MDSD_FLUENT_SOCKET_PORT" >> ~/.bashrc
             export ENABLE_MCS="true"
             echo "export ENABLE_MCS=$ENABLE_MCS" >> ~/.bashrc
+            export SSL_CERT_FILE="/etc/pki/tls/certs/ca-bundle.crt"
```

Comment:
why do we need this? is this for Mariner specific? or the HTTPs proxy related?

Reply:
This is needed to connect to MDSD from MAriner...It is not related to proxy